config.json

```diff
@@ -1090,6 +1090,39 @@
         "nugetId": "Xamarin.AndroidX.Wear",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.wear",
+        "artifactId": "wear-ongoing",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0.0",
+        "nugetId": "Xamarin.AndroidX.Wear.Ongoing",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear",
+        "artifactId": "wear-input",
+        "version": "1.1.0",
+        "nugetVersion": "1.0.0.0",
+        "nugetId": "Xamarin.AndroidX.Wear.Input",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear",
+        "artifactId": "wear-phone-interactions",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0.0",
+        "nugetId": "Xamarin.AndroidX.Wear.PhoneInteractions",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear",
+        "artifactId": "wear-remote-interactions",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0.0",
+        "nugetId": "Xamarin.AndroidX.Wear.RemoteInteractions",        
+        "dependencyOnly": false,
+        "excludedRuntimeDependencies": "com.google.android.gms.play-services-basement,com.google.android.gms.play-services-wearable"
```

Comment:
Why exclude these dependencies?  If they are needed at runtime, apps will crash when they try to use this package.  Since we already bind these, I think you need to add them as `"dependencyOnly": true` entries to this file.